README.md

```diff
@@ -176,7 +176,22 @@ nf.format(string);
 // Proposed: "987,654,321,987,654,321"
 ```
 
-We will reference existing standards for interpreting decimal number strings where possible.
+A common use case is to store currency amounts as a BigInt of a small unit, like cents. This API can be used to retain full precision of the BigInt:
+
+```javascript
+const nf = new Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "EUR",
+});
+const bi = 1000000000000000110000n;
+nf.format(bi + "E-6");
+// Current:  "€1,000,000,000,000,000.10"
+// Proposed: "€1,000,000,000,000,000.11"
+```
+
+The general syntax for decimal strings, essentially `#.#E#`, is a widely understood interchange format in computing. The specific version we use is the ECMA-262 `StringNumericLiteral` grammar, which also allows non-base-10 numbers like hexadecimal and binary.
```

Comment:
I don't know if I would call the E-notation `#.#E#` an interchange format, but I do agree it's widely understood notation.

Reply:
Sure, I removed the word "interchange".